client/app/index.html

```diff
@@ -5,6 +5,18 @@
     <meta charset="UTF-8">
     <base href="/">
     <title>Redash</title>
+    <script>
+      function onIE() {
+        window.location.href = '/unsupported';
+      }
+      if (
+        navigator.userAgent.match('/MSIE') || // IE10
+        navigator.appVersion.match('Trident/') // IE11
+      ){
+        onIE();
+      }
+    </script>
+    <!--[if IE]><script>onIE()</script><![endif]-->
```

Comment:
IE only for now, consider using https://github.com/browserslist/browserslist-useragent in the future.